.gitignore

```diff
@@ -3,3 +3,6 @@ docs/versions/devel.md
 
 # generated output from go run ./... compile
 checklist.md
+Gemfile
+Gemfile.lock
```

Comment:
Do we want to ignore these? Seems like it'd be beneficial for publishing (and for folks doing local builds, too)

Reply:
https://baseline.openssf.org/versions/2025-02-25#osps-qa-0201 suggests that we should probably have at least the Gemfile (and I'd recommend both)

Reply:
If the output (released artifact) is the website, then the Jekyll configuration (and the go templating) are both the software used to produce the output.  If that's not the intent, then we should figure out how to clarify.

On the other hand, the version of Jekyll used to generate the website is actually specified by https://github.com/ossf/security-baseline/blob/main/.github/workflows/web-publish.yml#L36, so maybe this is just "development cruft" that could be ignored as it only impacts contributors attempting to build a local copy.